src/tools/meson.build

```diff
@@ -1,11 +1,16 @@
+paktool_deps = [
+  nputil_dep,
+  np2D_dep,
+  phyfs_dep,
+  ]
 paktool = ['bmp2pak', 'pak2bmp']
 
 foreach tool : paktool
   executable(
     tool,
     tool + '.cpp',
     win_subsystem: 'console',
-    dependencies: (platform_is_windows ? netpanzer_dep_static : netpanzer_dep_shared),
+    dependencies: paktool_deps,
```

Comment:
@winrid Note here especially: we're no longer linking to the entire netpanzer library, but only to a few individual dependencies.

A small advantage in this case, but maybe more useful in the long-term.

cc @jammyjamjamman

Reply:
Actually I remember I noticed in the scons build there was linking to individual objects like that.